src/main/scala/it/pagopa/interop/agreementprocess/service/model/UserResponse.scala

```diff
@@ -2,4 +2,4 @@ package it.pagopa.interop.agreementprocess.service.model
 
 import java.util.UUID
 
-final case class UserResponse(email: String, id: UUID, name: String, surname: String, taxCode: String)
+final case class UserResponse(email: Option[String], id: UUID, name: String, surname: String, taxCode: String)
```

Comment:
The `email` is no more required
```suggestion
final case class UserResponse(id: UUID, name: String, surname: String, taxCode: String)
```